src/auth/auth.module.ts

```diff
@@ -6,6 +6,8 @@ import { JwtStrategy } from './jwt.strategy';
 import { ApiGuard } from './guards/api.guard';
 import { ConfigService } from '@nestjs/config';
 import { PrismaService } from '../prisma/prisma.service';
+import { MixedGuard } from './guards/mixed.guard';
```

Comment:
Codacy found an issue: ['MixedGuard' is defined but never used.](https://app.codacy.com/gh/Visual-Regression-Tracker/backend/pullRequest?prid=6005401)